core/src/test/scala/unit/kafka/zk/EmbeddedZookeeper.scala

```diff
@@ -37,6 +37,8 @@ class EmbeddedZookeeper() {
   factory.startup(zookeeper)
   val port = zookeeper.getClientPort()
 
+  println("Starting embedded zookeeper on port " + port + " using directories " + snapshotDir + "," + logDir)
```

Comment:
You can change the logging level in core/src/test/resources/log4j.properties (assuming your test in in core, other packages have their own log4j.properties file for their tests) when running the test locally, it is default to OFF so you won't see anything beside test failure trace.